docs/content/releases.md

```diff
@@ -6,6 +6,23 @@ doctypes: ["concept"]
 toc: true
 docs: "DOCS-616"
 ---
+## NGINX Ingress Controller 2.4.1
+
+19 Oct 2022
+
+CHANGES:
+* Update NGINX version to 1.23.2.
+* Update NGINX Plus version to R27 P1.
+* [3180](https://github.com/nginxinc/kubernetes-ingress/pull/3180) Remove all IPV6 listeners in ingress resources with -disable-ipv6 command line.
```

Comment:
Looks like there were two cherry picks now, maybe we should just link the two original PRs?